runtime/vm/jvminit.c

```diff
@@ -3546,6 +3546,10 @@ threadInitStages(J9JavaVM* vm, IDATA stage, void* reserved)
 				loadInfo->fatalErrorStr = "cannot parse -Xjni:";
 				return returnVal;
 			}
+
+			if ( 0 >= FIND_ARG_IN_VMARGS(EXACT_MATCH, "-XX:disableCPUAffinity", NULL)) {
```

Comment:
Can you define the new option in jvminit.h?  That's typically where options get defined so there's a single canonical spelling.

Reply:
We also typically want to consume args when they are used so that options like `-XX:-IgnoreUnrecognizedXXColonOptions` work correctly